### ISSUE TITLE
✨ CSS Context Provider for `@master/css.react`

### ISSUE THREAD

**Issue by itsezc** [master-co/css#123]
### Description

Prevent having the need to pass the `css` for every hook call, set default state and centralize your codebase and all while improving DX.

**Comment by 1aron** [comment]
It's expected to provide a Master CSS Context Provider that contains initialization of the JIT process and theme-related state.
```tsx
import { CSSProvider } from '@master/css.react'
import { config } from '../master.css.js'

function App() {
    return (
        <CSSProvider config={config}>
            ....
        </CSSProvider>
    )
}

export default App
```

```tsx
import { CSSContext } from '@master/css.react';

export default function ThemeButton({ onChange, className }: any) {
    const { theme, changeTheme } = useContext(CSSContext)
    ...
}
```

**Comment by 1aron** [comment]
Released at [v2.0.0-beta.118](https://github.com/master-co/css/releases/tag/v2.0.0-beta.118)
